dendrite/pipeline.yml

```diff
@@ -25,6 +25,7 @@ steps:
       - docker#v3.0.1:
           image: "golang:1.15"
           mount-buildkite-agent: false
+          propagate-environment: true
```

Comment:
maybe too late now, but some comments as to why this is needed might have been been useful. (Or, yes, moving the `env` settings onto the docker plugin)

Reply:
Since `propagate-environment` is used in other steps, I didn't think that was necessary.
[Docs](https://github.com/buildkite-plugins/docker-buildkite-plugin/tree/v3.9.0#propagate-environment-optional-boolean) for this. :)

Reply:
mmm, fair.